Shopist/Shopist.xcodeproj/project.pbxproj

```diff
@@ -7,10 +7,14 @@
 	objects = {
 
 /* Begin PBXBuildFile section */
+		61441B84245B15AF003D8BB8 /* DebugTracingViewController.swift in Sources */ = {isa = PBXBuildFile; fileRef = 61441B83245B15AF003D8BB8 /* DebugTracingViewController.swift */; };
+		61441B8C245B20DF003D8BB8 /* UIButton+Disabling.swift in Sources */ = {isa = PBXBuildFile; fileRef = 61441B8B245B20DF003D8BB8 /* UIButton+Disabling.swift */; };
+		617CEB41245719E200AD4669 /* OpenTracing.framework in Frameworks */ = {isa = PBXBuildFile; fileRef = 617CEB3C245719E200AD4669 /* OpenTracing.framework */; };
+		617CEB42245719E200AD4669 /* OpenTracing.framework in Embed Frameworks */ = {isa = PBXBuildFile; fileRef = 617CEB3C245719E200AD4669 /* OpenTracing.framework */; settings = {ATTRIBUTES = (CodeSignOnCopy, RemoveHeadersOnCopy, ); }; };
```

Comment:
it seems like we embed `OpenTracing` twice: once into `Datadog.framework` and once into `Shopist.app`
could we find a better way to share one `OpenTracing.framework` between 2 targets?

Reply:
As discussed on Slack, we spot a wider issue of delivering the `OpenTracing.framework` dependency for `Datadog.framework`. This fix is detached and will be done in `RUMM-432`.